src/Css.elm

```diff
@@ -1454,20 +1556,20 @@ introduceSelector selector declarations =
             [ StyleBlock (SingleSelector selector) [] [] ]
 
         {- If there are no properties declared, then we have a situation
-            like `Bar` in the following:
+                            like `Bar` in the following:
 
-                stylesheet "homepage"
-                    . Foo . Bar
-                        ~ fontWeight bold
+                                stylesheet "homepage"
+                                    . Foo . Bar
+                                        ~ fontWeight bold
 
-                ...as opposed to:
+                                ...as opposed to:
 
-                stylesheet "homepage"
-                    . Foo
-                        ~ fontWeight bold
+                                stylesheet "homepage"
+                                    . Foo
+                                        ~ fontWeight bold
 
-            In this case, we don't make a new declaration, but rather add
-            the new selector to the existing list of selectors.
+                            In this case, we don't make a new declaration, but rather add
+                            the new selector to the existing list of selectors.
```

Comment:
I thought it looked kind of odd! But I know there have been some changes and discussion around elm-format's style on elm-discuss recently so I wasn't sure. I'll keep an eye out for this in the future.